packages/openapi-ts/src/getLogs.ts

```diff
@@ -15,3 +16,13 @@ export const getLogs = (userConfig: UserConfig | undefined): Config['logs'] => {
   }
   return logs;
 };
+
+export const getLogsFileInUse = (
```

Comment:
This shouldn't be needed, there's a single function for defining the whole logs object so you can safely use it across the application

Reply:
Ah yeah kinda overlooked this. Fixing..